web/src/engine/websites/MadaraDex.ts

```diff
@@ -0,0 +1,55 @@
+import { Tags } from '../Tags';
+import icon from './MadaraDex.webp';
+import { type Chapter, DecoratableMangaScraper, Page, type MangaScraper } from '../providers/MangaPlugin';
+import * as Madara from './decorators/WordPressMadara';
+import { Fetch, FetchCSS, FetchRequest } from '../FetchProvider';
+import type { Priority } from '../taskpool/TaskPool';
+import DeProxify from '../transformers/ImageLinkDeProxifier';
+
+function ChapterPageExtractor(this: MangaScraper, image: HTMLImageElement): string {
+    const url = image.dataset?.src || image.dataset?.lazySrc || image.srcset || image.src;
+    const uri = new URL(url.trim(), this.URI);
+    return uri.href;
+}
+
+@Madara.MangaCSS(/^{origin}\/title\/[^/]+\/$/, 'meta[property="og:title"]:not([content*="Read "])')
+@Madara.MangasMultiPageAJAX()
+@Madara.ChaptersSinglePageAJAXv2()
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('madaradex', 'MadaraDex', 'https://madaradex.org', Tags.Media.Manhwa, Tags.Language.English, Tags.Rating.Pornographic);
```

Comment:
Should this be tagged as `Aggregator`?
Other websites as well...

Reply:
If website isnt "Official" or "Scanlator" should be consider it "Aggregator" by default? 🤷 

I wonder.